packages/select/src/components/FzSelectLabel.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <button
+    disabled
+    test-id="fzselect-label"
+    :title="option.label"
+    :class="computedClass"
+  >
+    {{ option.label }}
+  </button>
```

Comment:
why a button element for a label?

Reply:
to interrupt the click event. Label or span elements with the disabled attribute do not work. I also tried blocking the click event with some ‘@click.stop’/‘@click.prevent’ modifiers, but they did nothing.